operations/helm/charts/alloy/templates/cluster_service.yaml

```diff
@@ -4,6 +4,7 @@ apiVersion: v1
 kind: Service
 metadata:
   name: {{ include "alloy.fullname" . }}-cluster
+  namespace: {{ include "alloy.namespace" . }}
```

Comment:
@valorl excuse my ignorance, but why use `alloy.namespace` instead of `Release.Namespace`?